### ISSUE TITLE
Add support of building cosolvent simulations that have a membrane and cosolvents but no protein

### ISSUE THREAD

**Issue by allisonbarkdull** [forlilab/cosolvkit#19]

**Comment by nbruciaferri** [comment]
I was just working on this issue and It looks like the problem is more deep.
Apparently OpenMM's Modeller doesn't allow the system to create a membrane without having a protein.

We should discuss more this issue, what possible alternative approaches can we use and what would be the pros and cons.

Including @diogomart, @mallanos and @mattholc in the conversation to get their perspective.

**Comment by mallanos** [comment]
I implemented something similar for Allison's case in Autopath, to build a ligand-membrane system. First created the Modeller from the ligand and then added the membrane. If you just want membrane+cosolvent and no ligands, maybe we could create a dummy atom and build the membrane from there. We can discuss this further

**Comment by nbruciaferri** [comment]
Yeah that's the solution I came up with as well, I just think it's not very elegant XD .
